### ISSUE TITLE
MITMProxy does not start on macOS

### ISSUE THREAD

**Issue by renyard** [jvilk/mitmproxy-node#6]
I'm trying to run BLeak on macOS, but it fails to start MITMProxy through mitmproxy-node. It appears that `waitForPort(8080, 1);` at src/index.ts:361 resolves successfully even though MITMProxy is not running.

I have confirmed that there is nothing listening on port 8080:
```shell
➜  ~  lsof -nP -i4TCP:8080 | grep LISTEN
➜  ~
```

macOS: 10.13.6
Node: 8.15.1
Python: 3.7.2

**Comment by Jonahss** [comment]
Works for me.

macOS: 10.14.3
Node: 11.6.0
Python: 3.7.2

**Comment by mishackerr** [comment]
@renyard did you resolve the issue?

**Comment by TJMaria** [comment]
I'm also having issues with getting BLeak to run on MacOs, so very curious to know if you resolved the issue @renyard

**Comment by renyard** [comment]
@TMakaku @MichaelGuoXY No, I never did resolve the issue to get it running on macOS directly. I ended up installing it in a Linux VM instead.
Could be related to macOS or Node versions though, as I see it works for @Jonahss on newer versions of both.

**Comment by TJMaria** [comment]
@renyard Thanks for the reply. Trying it on linux was next on my list.
